Examples/Tests/scraping/inputs_rz

```diff
@@ -51,3 +52,6 @@ diag2.intervals = 1
 diag2.diag_type = Full
 diag2.fields_to_plot = Er
 diag2.format = openpmd
+
+diag3.diag_type = BoundaryScraping
```

Comment:
Please add in `Regression/WarpX-tests.ini`
```
addToCompileString = ... USE_OPENPMD=TRUE
cmakeSetupOpts = ... -DWarpX_OPENPMD=ON
```
so we can filter the test out if we build w/o openPMD.